doc/ref_arch/openstack/chapters/chapter08.rst

```diff
@@ -436,9 +415,8 @@ Image - Glance API testing
 ''''''''''''''''''''''''''
 
 Glance API is covered in the OpenStack Gates via
-`Tempest <https://opendev.org/openstack/tempest>`__ as integrated in
-`Functest Smoke
-CNTT <https://git.opnfv.org/functest/tree/docker/smoke-cntt/testcases.yaml>`__.
+:cite:p:`rally` as integrated in
```

Comment:
the original is "Tempest" -- if changing to "rally" then maybe we need a note as to the reason for the change.

Reply:
@pgoyal01 well catch. cited is wrong here. Thx

Reply:
Fixed in next commit